types/office-js-preview/index.d.ts

```diff
@@ -1946,23 +1535,6 @@ declare namespace Office {
     }
     /**
      * Specifies the type of the XML node.
```

Comment:
Shouldn't this one include pointing folks to the section on the CustomXmlParts reqt. set?